pkg/monitor/worker.go

```diff
@@ -198,10 +230,15 @@ func (mon *monitor) workOne(ctx context.Context, log *logrus.Entry, doc *api.Ope
 		return
 	}
 
-	hiveRestConfig, err := mon.liveConfig.HiveRestConfig(ctx, 1)
-	if err != nil {
-		// TODO(hive): Update to fail once we have Hive everywhere in prod and dev
-		log.Info(err)
+	// in the future we will have the shard index set on the api.OpenShiftClusterDocument but for
+	// now we simply select Hive (AKS) shard 1 because we only have one and sharding is yet to come
+	// e.g. shard := mon.hiveShardConfigs[doc.shardIndex]
+	shard := 1
```

Comment:
seems to be somewhat duplicated with line 100

Reply:
Intentionally duplicated so the reader is aware of why the shard is being set to 1 here and also marks the place where we can come and change it to  `mon.hiveShardConfigs[doc.shardIndex]`. Do you really think it needs to be removed or just reworded to not be a copy?

Reply:
I modified the comment after adding the accessor functions. Have a look in the next round and let me know if it still looks duplicated.

Reply:
we could probably have shard as a package const so we don't need to look for every instance of that pattern when we remove it, just delete the const and fix all the errors instead